src/helpers/email.helper.ts

```diff
@@ -0,0 +1,44 @@
+import * as nodemailer from 'nodemailer';
+import { logger } from '../helpers/logger';
+
+export const sendImageEmailNotification = (
+  emails: string[],
+  displayName: string,
+  imageId: number,
+) => {
+  const getTrans = () => {
+    return nodemailer.createTransport({
+      host: process.env.MAILER_HOST,
+      port: 587 /*process.env.MAILER_PORT*/,
+      secure: false /*process.env.MAILER_SECURE*/,
```

Comment:
I changed it, but then it makes problems, Transport does not recognize type of it (port and secure) and makes "host" get red. It expects number and boolean, or smpt in that place.

Reply:
That is because all .env constant are type string, you have to convert to integer and boolean.

Reply:
Thanks, fixed. For number I found two ways, both works: Number( ) or parseInt( ). Maybe I could put Number(), but parseInt() looks beter.
For boolean is different, to make it work must put Boolean( ) and in .env file leave it empty.  Empty means that the value of boolean is false. (MAILER_SECURE=) or (MAILER_SECURE=""), thats it, if you put something in it, compailer will read true (because its boolean).